### PR TITLE
Feat : Page note found page

### PR DESCRIPTION
**Pull request by anuj-kumary**
Added 404 Error Page
Preview: https://deploy-preview-15--eaglestores-react.netlify.app/

### CODE REVIEW

**Review by Rohanmond — APPROVED** [review]
code looks good to merge

**Review by znjs — APPROVED** [review]
- Clean code
- Good naming conventions used

File: eagle-stores/src/pages/PageNotFound/PageNotFound.jsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      <div className='container'>
+        <div className='container__left'>
+          <img
+            src='https://s3-us-west-2.amazonaws.com/s.cdpn.io/4424790/Mirror.png'
+            alt='Error Page'
+          />
+        </div>
+        <div className='container__right'>
+          <h3 className='heading text__center'>Awwww... Don't Cry</h3>
+          <h5 className='sub__heading text__center'>It's just a 404 Error!</h5>
```
Comment (hinanshisuthar) [inline eagle-stores/src/pages/PageNotFound/PageNotFound.jsx:16]: Good error messages

**Review by hinanshisuthar — APPROVED** [review]
- Clean code practices followed
- Good error message given
- 404 page designes instead of keepint it as blank

**Pull request merged**